### PR TITLE
Tuya Fan Support

### PR DESCRIPTION
**Pull request by gslender**
This adds Fan device/driver support for the Tuya App

### CODE REVIEW

File: lib/TuyaOAuth2DriverFan.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const TuyaOAuth2Driver = require('./TuyaOAuth2Driver');
+const TuyaOAuth2Constants = require('./TuyaOAuth2Constants');
+
+/**
+ * @extends TuyaOAuth2Driver
+ * @hideconstructor
+ */
+class TuyaOAuth2DriverLight extends TuyaOAuth2Driver {
```
Comment (WeeJeWel) [inline lib/TuyaOAuth2DriverFan.js:10]: Can you rename this to `TuyaOAuth2DriverFan`?

File: drivers/fan/pair/select_country.html [hunk 2]
```diff
@@ -0,0 +1,160 @@
+<style type="text/css">
```
Comment (WeeJeWel) [inline drivers/fan/pair/select_country.html:1]: This file is not used anymore. Can you remove it?

**Review by WeeJeWel — CHANGES_REQUESTED** [review]
WOW! Now this is how I like PRs! 😍 

I only have added two small remarks. Could you solve those? Then we'll merge & publish this PR.

Again, thank you!!

**Comment by gslender** [comment]
Changes made and should be ready to be accepted

**Comment by WeeJeWel** [comment]
Amazing, thank you!

**Pull request merged**